app/views/vendors/index.html.erb

```diff
@@ -1,6 +1,11 @@
 <div class="vendors index_vendor_page col-sm-12">
 
-  <%= render :partial => 'vendor_alert' %>
+  <% if flash[:notice] %>
+    <div class = "alert alert-<%= flash[:notice_type] || 'info' %>" >
+      <%= flash[:notice] %>
+      <a href = "#" class = "close" data-dismiss = "alert" aria-label = "close">&times;</a>
+    </div>
+  <% end %>
```

Comment:
Could the validation alert partial be written more generally so that it could be used here too?